doc/docportal/use_scummvm/taskbar_integration.rst

```diff
@@ -0,0 +1,46 @@
+===========================
+Taskbar integration
+===========================
+
+The taskbar integration has several features:
+
+- Adds an overlay icon in the taskbar/dock when running a game (Windows and macOS)
+- Updates a list of recently played games that can be started from the taskbar/dock (Windows and macOS).
+- Shows a progress bar when doing a mass add of games, and shows the number of games found at the end (Windows, macOS and Linux).
+- Shows an error state in the taskbar if an error occurs when running a game (Windows only).
+
+.. _iconspath: 
+
+Setting up the icon overlay
+================================
+
+Icons for many games can be found on our `Github repository <https://github.com/scummvm/scummvm-icons>`__. 
+
+Place the icons in a folder, and either change the *iconspath* in the :doc:`configuration file <../advanced_topics/configuration_file>` to point to this folder, or add the folder to the :ref:`Extra Path <extra>` on the Paths settings tab.
+
+The icon files must be ICO files for Windows, or PNG files for macOS. They must follow one of these naming conventions, where xxx is the file extension:
+
+		- gameid.xxx
+		- :ref:`ID <id>`.xxx
+		- engineid-gameid.xxx
+
+To find the gameid and the engineid, open the :doc:`configuration file <../advanced_topics/configuration_file>`. The gameid is the name of the game in square brackets. The game entry will be similar to this:
```

Comment:
This is incorrect. Both "gameid" and "engineid" are listed under the entry. The name in square brackets [] is the (target) ID, which may or may not match the gameid.

Reply:
Thank you - fixed.